source/fundamentals/linq.txt

```diff
@@ -497,6 +497,82 @@ The following is a subset of the returned results:
        ]
     }
 
+$vectorSearch
+~~~~~~~~~~~~~
+
+The ``$vectorSearch`` aggregation stage performs an Approximate Nearest Neighbor search
```

Comment:
S: It looks like Approximate Nearest Neighbor isn't usually capitalized. [SG recommends italics for new terms](https://www.mongodb.com/docs/meta/style-guide/style/text-formatting/#:~:text=HTTP%20check%20box.-,Terms%2C%20new,-Format%20using%20italics)

```suggestion
The ``$vectorSearch`` aggregation stage performs an *approximate nearest neighbor* search
```